TGM/src/main/java/network/warzone/tgm/command/CycleCommands.java

```diff
@@ -432,36 +432,36 @@ public static void loadmaps(CommandContext cmd, CommandSender sender) {
     @Command(aliases = {"channel", "chatchannel", "cc"}, desc = "Change or select a chat channel.", usage = "(all|team|staff)", min = 1)
     public static void channel(CommandContext cmd, CommandSender sender) {
         if(!(sender instanceof Player)) {
-            sender.sendMessage("Error: Only players can use this command.");
+            sender.sendMessage("Only players can use this command.");
             return;
         }
         Player player = (Player) sender;
         if (cmd.argsLength() == 0) {
-          player.sendMessage(ColorConverter.filterString("&cUsage: /channel (name)"));
+          player.sendMessage(ChatColor.RED + "Usage: /channel (name)");
           return;
         }
 
         String channelName = cmd.getString(0).toUpperCase();
         ChatModule.Channel channel = ChatModule.Channel.byName(channelName);
         if (channel == null) {
-          player.sendMessage(ColorConverter.filterString("&cInvalid channel: " + channelName));
+          player.sendMessage(ChatColor.RED + "Invalid channel: " + channelName);
           player.sendMessage(ColorConverter.filterString("&cChannels: ( " + StringUtils.join(Arrays.stream(ChatModule.Channel.values()).filter(ch -> ch.hasPermission(player)).collect(Collectors.toList()), " | ")) + " )");
           return;
         }
 
         if (!channel.hasPermission(player)) {
-          player.sendMessage(ColorConverter.filterString("&cError: Insufficient permissions."));
+          player.sendMessage(ChatColor.RED + "Insufficient permissions.");
```

Comment:
https://github.com/WarzoneMC/Warzone/pull/387/commits/7bf7a1605e505c2c52e19b985c6d7d0e4780f22d#r230930294

Reply:
too long uneeded code

Reply:
In some scenarios you're actually making it longer, because ChatColor.XXX is way longer then &x.

Reply:
I still believe `filterString(...)` is fine. The reason of "too long unneeded code" doesn't really matter. Shorter lines of code doesn't necessarily mean it's better/more efficient than something else.